src/sentry/utils/sdk.py

```diff
@@ -401,15 +403,21 @@ def _capture_anything(self, method_name, *args, **kwargs):
                     relay_envelope.items = envelope.items.copy()
                     args = (relay_envelope, *args_list[1:])
 
-                if is_current_event_safe():
-                    metrics.incr("internal.captured.events.relay")
-                    getattr(sentry_saas_transport, method_name)(*args, **kwargs)
-                else:
-                    metrics.incr(
-                        "internal.uncaptured.events.relay",
-                        skip_internal=False,
-                        tags={"reason": "unsafe"},
-                    )
+                if sentry_saas_transport:
+                    if is_current_event_safe():
+                        metrics.incr("internal.captured.events.relay")
+                        getattr(sentry_saas_transport, method_name)(*args, **kwargs)
+                    else:
+                        metrics.incr(
+                            "internal.uncaptured.events.relay",
+                            skip_internal=False,
+                            tags={"reason": "unsafe"},
+                        )
+
+                if experimental_transport:
+                    if is_current_event_safe():
```

Comment:
You can do two things with the experimental transport:
1. Either use it as intended and mark transactions you want to try so they trip `is_current_event_experimental` and roll out incrementally, in which case you also want to add `is_current_event_experimental()` here, so you filter.
2. Just roll it all out to experimental with your new option, which is probably simpler

I think 2 sounds fine if you're careful to slowly roll out. 

Either way you also need to remember to remove the return here so it doesn't exit early, since it was previously an "either or" to send events to the experimental dsn or saas, but you really want to be "both"
https://github.com/getsentry/sentry/pull/65719/files#diff-0567d850bc5a9cd5395cab9dec6fd5cd6b76f54db06c53d98a551aa278e11e0aL357-L361

Reply:
Yeah, I was intending to do option no. 2 with this change!

My intent was to support both behaviours: only send to experimental dsn for explicitly marked "experimental" events or send to both SaaS and experimental otherwise. 

That early return is triggered only if `is_current_event_experimental()` (ie has the experimental tag set in the scope), in which case we would want it to early return and not send it to SaaS as was the intended behaviour. If it's not explicitly marked as experimental, it gets sent to both.

If we don't want to allow sending "experimental" events (since we don't send any right now), I can just remove that whole if block you linked.

Reply:
(@k-fish)

Reply:
I also tested it locally with the rollout set to 0 vs 1

![Screenshot 2024-02-23 at 2 27 44 PM](https://github.com/getsentry/sentry/assets/63818634/20c043ef-8c51-48f5-9544-ca2d1b0e8f6e)

Reply:
> If we don't want to allow sending "experimental" events (since we don't send any right now), I can just remove that whole if block you linked.

Yep just delete that part, no one is using it right now afaik

Reply:
I missed that part w/ only early returning, so it only applies to 1. above, 2. looks fine 🚢